django/utils/http.py

```diff
@@ -293,12 +295,21 @@ def is_safe_url(url, host=None):
             url = force_text(url)
         except UnicodeDecodeError:
             return False
+    if allowed_hosts is None:
+        allowed_hosts = set()
+    if host:
+        warnings.warn(
```

Comment:
@MarkusH, Any reason not to use `allowed_hosts` for the new kwarg? I understand why we'd want to keep `host` around but `hosts?` doesn't strike me as a particularly precise name while `allowed_hosts` describes exactly what it's used for and is along the lines of the `ALLOWED_HOSTS` parameter.